app/Http/Controllers/HomeController.php

```diff
@@ -68,4 +69,61 @@ public function getPicture($filename){
 
         return $response;
     }
+
+    /* Report bug */
+    public function indexReportBug()
+    {
+        return view('report_bug');
+    }
+
+    public function reportBug(Request $request)
+    {
+        $user = Auth::user();
+        if(Auth::user()){
+            $username = htmlspecialchars(stripslashes(Auth::user()->name));
+        } else {
+            $username = 'an unauthenticated user';
+        }
+
+        //personal auth token from your github.com account
+        $token = env('GITHUB_AUTH_TOKEN');
+
+        $url = "https://api.github.com/repos/luksan47/mars/issues";
+
+        //request details, removing slashes and sanitize content
+        $title = htmlspecialchars(stripslashes('Reported bug'), ENT_QUOTES);
```

Comment:
Is `htmlspecialchars` needed there? (I'm not a 100% familiar with this)

Reply:
If it's a POST, it should not be needed. I will test is.

Reply:
Not sure, but I think we need it based on testing different scenarios.

Reply:
Okay, let's keep it, it does not hurt.